src/pages/Home.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import '../App.css';
+
+function Home() {
+  return (
+    <div className="home">
+      <h2 className="homeHeader">Welcome to our page!</h2>
+      <p className="homePara">
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Sequi nam
+        cumque similique quidem nemo alias iusto officia, unde doloribus iste
+        suscipit vel omnis quis fugiat provident quibusdam nobis rem dolor
+        assumenda aut? Sed ipsa nam molestiae molestias saepe inventore! Velit!
+      </p>
+
+      <p className="homePara">
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Sequi nam
+        cumque similique quidem nemo alias iusto officia, unde doloribus iste
+        suscipit vel omnis quis fugiat provident quibusdam nobis rem dolor
+        assumenda aut? Sed ipsa nam molestiae molestias saepe inventore! Velit!
+      </p>
+    </div>
+  );
```

Comment:
[OPTIONAL] I think it will be better if you can add content about you or your project instead of using lorem ☺️ 😌